lib/active_admin/views/components/paginated_collection.rb

```diff
@@ -94,7 +94,7 @@ def build_per_page_select
       end
 
       def build_pagination
-        options = {}
+        options = {theme: 'active_admin'}
```

Comment:
Space inside { missing.<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.